### ISSUE TITLE
[RFC] GT Solar Panels to strong

### ISSUE THREAD

**Issue by alphaNOVAircraft** [GTNewHorizons/NewHorizons#3232]
#### Which modpack version are you using?
2.0.4.6
#
#### If in multiplayer; On which server does this happen? 
Private
#
#### Why im writing this issue?
Atm we are in the middle of Tier 6, in order to proceed that far, we used a ic2 fluid reactor, it is definitly worth the energy (producing around 30k eu/t with 3 quar uranium rods). It takes a good ammount of space and definitly feels balanced. 

Due to the fact that we got ourselfs the resources to get 4 IV solars and the fact we are in a void world, we can produce 32k eu/t passive, it just feels way to strong for the amount of resources we needed to make those. A reactor (also not a cheap thing) takes nearly a whole chunck for producing less energy and you even need some fuel for it. 
#

#### What do you suggest instead/what changes do you propose?
I would nerf the solars a bit, e.g. just add some more circuits to the recipe. Atm you need two circuits of two tiers higher than the solar you want to craft (you want an hv solar, you need 2 IV circuits). That changes as soon as you reach LuV solars, there you need 4 UV solar panels. 

I would suggest that you need from the moment you have to use the dire crafting table (HV), the recipes should all use 4 circuits, not only 2. Such high tier circuits need some affort to get and it will be still worth in my opinion.

**Comment by tranceplanet** [comment]
I personally don't agree with this.  I believe solar is very challenging to make at early levels.  Fuel/Steam/Turbines and other power are still the methods people use early on.  The recipes for solar are already 1 tier behind (I'm EV and I would only just be able to make HV solars - making lots of HV solars isn't appealing for powering my base, maybe just running a remote oil drill or something).  I have a lot of work to go before I can even make EV solar, by that time I'll be midway through IV and using other power.

Maybe this is an end-game perspective but from an early/midgame perspective, they aren't strong.

**Comment by 0lafe** [comment]
A problem is that you're comparing ic2 fluids to 4IV solars. But yes, IV solars in general are a little on the good side and should probably get some kinda Nerf. I made 16 when I was in IV and that brought me to fusion. 

I think for the most part, as long as you can't make solars of the tier you're in, they're ok. Maybe change the IV circuit one, and I'd also love if UVs took less glowstone. That was always the part that discouraged me from them. Maybe make them take more of something else that needs ebfing and stuff.

**Comment by Vicus-N** [comment]
So much time need to be invested in this solars, so I'm in early EV is not ever thinking about solar power... I'll better build Nuclear or Wind gens...

**Comment by bartimaeusnek** [comment]
since they got the new Superconductors in their recipe, i think they are balanced as they are now, maybe add another 2 circuits per solar from EV+ on but dunno about that

**Comment by boubou19** [comment]
don't forget the fact that you still need to clean them with water sometimes

**Comment by boubou19** [comment]
then nerf the block versions because those are really passive

**Comment by alphaNOVAircraft** [comment]
you can clean them by simply get the cover and add it again, takes you 1 sek, after one day of usage they still produe around 6 to 7 keu/t. i would not nerf the block version of them.

**Comment by alphaNOVAircraft** [comment]
> So much time need to be invested in this solars

Not really, it took me around 4 hours to build them (beginning at the 1 eu one)

> making lots of HV solars isn't appealing for powering my base

Sure, it wasnt appealing for me back in HV, back then it is still better to use oil e.g. because it is really strong back then. but that changes when you reach IV, until then oil was just fine for me.

My problem with it is simply the balance between the good power sources in IV, one one hand you have a fluid reactor (a good choice at the beginning but uses up some space) 
![2018-05-16_12 37 25](https://user-images.githubusercontent.com/29660362/42137163-f46181ae-7d67-11e8-9417-ef27ba1e52d0.png)
![2018-05-22_19 18 18](https://user-images.githubusercontent.com/29660362/42137167-04873cae-7d68-11e8-8c7b-85f9a73a13e7.png)
That one is the setup im using for my uranium power, it takes a decent amount of space and needs 3 quad rods every 5 n half hours. Just does not feel balanced compared to the solars

Especially in combination with the GT++ energy sub station, solar is just insanly strong, you dont use the full 32k eu/t most of the time, but you can eaily store 80 billion eu in one of those, so it can feed your base with way higher energy peaks if you need them.

**Comment by Prewf** [comment]
I think they're in an OK spot right now. You shouldn't really be comparing IV solars to fluid reactors imo. You have to consider that you're only able to "easily" make solar panels **because** you have a properly set up fluid reactor. If you didn't have the reactor, you would not have solar (aka you can't just rush to get the solar simply because the power requirements gate you from making one). 

By the same logic, you should want to nerf naquadah reactors too, because it's so easy to make and compact compared to an ic2 fluid reactor. But again, you can't make naquadah reactors easily without already having a readily available and stable source of IV/LuV power (you need a lot of zpm chips). It's just a natural consequence of having tiered up with a stable power source. You have an abundance of power and ease in automating things (ae2), of course it's going to feel like it's "easy" to make more power.

I feel like adding more chips to the HV/EV recipes isn't going to deter players from making them any more than it is right now. I understand the intention of your nerf isn't to deter players, but to instead add more challenge to the recipe. But honestly, in practice, it would only act as further deterrence on top of the little to no interest people have in building solars at that tier. The best you could do is add more chips to the IV+ solars, but you also have to consider that at this stage, players have AE2 set up properly (hopefully), so would adding more chips really make it more challenging? At IV and above, you have to think less about "difficulty" and more in terms of a resource/time sink. Instead of adding more chips, what about adding more of the new superconductor wires? And that's only IF you actually think IV+ solars deserve a nerf. imo they don't.

**Comment by alphaNOVAircraft** [comment]
> you should want to nerf naquadah reactors
 no definitly not, those are in a sweet spot, but you mentioned one difference between them
They take already 8 circuits

BUT the IV solar takes 2 ZPM ones, so i think changing that to 4 circuits would not harm to much by still making it harde to get more of them. 

I´m not interested in making solars useless but you should not be able to make a lot of them without needing a ton of resources, so it seemed to be the best way to double up the circuit price imo.

**Comment by Prewf** [comment]
>BUT the IV solar takes 2 ZPM ones, so i think changing that to 4 circuits would not harm to much by still making it harde to get more of them.

I guess that's fair. Might as well make it consistent with the existing alternatives.

**Comment by alphaNOVAircraft** [comment]
We maybe could start doubleing up the circuits in EV then, if you think HV is to early, i just want to make it more balanced also looking at the naquadah reactor, due to the fact there is an ev naquadah reactor, we might make the recipe of the EV solar more expensive, but dont touch the HV one.

**Comment by CalterOK** [comment]
Why not transfer the craft of high-level solar panels to the assembly line instead of craft table?

**Comment by alphaNOVAircraft** [comment]
> Why not transfer the craft of high-level solar panels to the assembly line instead of craft table?

Transfering stuff behind the assembly line is just the most extrem way of nerfing it, Just to get the assembly line you need over 100 zpm circuits, and i definitly think that that nerf would be way to hard. 

I suggested a small increase of the price of the solar panel, not a massiv nerf xD

**Comment by Prewf** [comment]
>Why not transfer the craft of high-level solar panels to the assembly line instead of craft table?

Parts of the high tier solar panels (LuV and above) are already gated behind an assembly line. There's little point in making the entire thing an assembly line except for nerfing for the sake of nerfing.

**Comment by CalterOK** [comment]
At the stage of the ME network only craft of the assembly line can cause difficulties, all the rest is a matter of crafting time.

**Comment by noobyaran9** [comment]
Honestly speaking, solar a little on the cheap side after you have everything ready for them. At least for those that are pre-ZPM solars. ZPM and UV solar a little to expensive to actually spam them due to the amount of resource and time sink on making them. If the nerf were to happen, please don't touch any of the solar that are above ZPM.

**Comment by NeroOneTrueKing** [comment]
Solars as a whole do not need a nerf. The **IV solar** is an anomaly.

Take a look at the recipe for the IV solar, specifically, the Irradiant Reinforced Chrome Plates in its recipe.
Look at the recipe for those:
![image](https://user-images.githubusercontent.com/28197216/42192419-69b9ee66-7e2e-11e8-8374-ad1e29be34f8.png)
They use a Quadruple Sunnarium Plate, which is super cheap compared to what's used in the EV solar's plate (a Sunnarium Alloy, which takes a half stack of iridium to make).
Change the Quadruple Sunnarium Plate in its recipe to an Enriched Sunnarium Alloy (like what's used in the LuV solar's plates) and it will cost a much more appropriate amount for the tier of power it provides.

**Comment by noobyaran9** [comment]
>Sunnarium Alloy

May i have a look at what this recipe is?

**Comment by NeroOneTrueKing** [comment]
It is:
4 sunnarium plates
8 reinforced iridium alloy plates
in the assembler:
![image](https://user-images.githubusercontent.com/28197216/42193296-60ec8046-7e33-11e8-8be7-c5e228b286ad.png)


Each reinforced iridium alloy plate is:
1 industrial diamond (has to be industrial, not regular)
4 advanced alloy
4 iridium plates
In the implosion compressor:
![image](https://user-images.githubusercontent.com/28197216/42193324-88cdfb3a-7e33-11e8-8049-3d2e186fb3a0.png)
![image](https://user-images.githubusercontent.com/28197216/42193309-7cbb9604-7e33-11e8-9f27-c875c3bbf8d9.png)

**Comment by noobyaran9** [comment]
I think just swap the Quadruple Sunnarium Plate and the Sunnarium alloy around the 2 recipe should fix this issue?

**Comment by 0lafe** [comment]
Mostly, but IV is still too cheap is the thing. I think that IV should need LuV circuit maker. Generally I'd like if they needed the tier above to make. This means IV needs assembly line, which is totally reasonable imo. Although, semi-related, I really think that the assembly parts were changed to cost too little eu/t. I think each tier should cost that tier to make. So LuV takes LuV and so on. It helps for gating them this way. Kinda complicated for UHV+

**Comment by noobyaran9** [comment]
>Mostly, but IV is still too cheap is the thing. I think that IV should need LuV circuit maker.

I thought it takes LuV circuit to make??
I think the circuit cost is fine, as @NeroOneTrueKing the reinforced plate is just too cheap for the solar panel. Changing the recipe of the reinforce plate to enriched sunnarium alloy should fix this issue.


> I really think that the assembly parts were changed to cost too little eu/t. I think each tier should cost that tier to make.

This is a really bad idea. The assembling line is already very expensive to make compared to the old recipe.

**Comment by 0lafe** [comment]
>I thought it takes LuV circuit to make??

I mean you need an LuV assembler. The new circuit tiers don't really follow machines enough for me to use circuit tiers to describe what I'm trying to say. I think solars should need the tier up assembler to make. Which for IV just happens to mean assembly line, which is perfect because wrong sunnarium thing asside, it was the most OP solar

**Comment by noobyaran9** [comment]
>I mean you need an LuV assembler. The new circuit tiers don't really follow machines enough for me to use circuit tiers to describe what I'm trying to say. I think solars should need the tier up assembler to make. Which for IV just happens to mean assembly line, which is perfect because wrong sunnarium thing asside, it was the most OP solar

Instead of modifying the circuit, why not we just change something in the recipe such that it uses LuV assembler to craft?? Circuit already requires the previous tiers to make unless you're in ZPM energy tier which allows u to create LuV circuit directly.

**Comment by 0lafe** [comment]
>Instead of modifying the circuit, why not we just change something in the recipe such that it uses LuV assembler to craft?? Circuit already requires the previous tiers to make unless you're in ZPM energy tier which allows u to create LuV circuit directly.

You're telling me that forcing people to make the circuit that requires 4 base ones, for an infinite amp of IV, is unreasonable!? I'm realising that I'm now interested as to why it's not always forcing the hardest circuit you can make, and doesn't that also mean you can make the IV one in EV?

**Comment by noobyaran9** [comment]
>I'm realising that I'm now interested as to why it's not always forcing the hardest circuit you can make, and doesn't that also mean you can make the IV one in EV?

Because it requires machine that are 2~3 tiers above the solar panel.

**Comment by 0lafe** [comment]
>Because it requires machine that are 2~3 tiers above the solar panel.

So what I want is for the IV solar to be the hardest solar to make with LuV assembler, and HV with hardest EV, and so on. Is that unreasonable?

**Comment by noobyaran9** [comment]
>So what I want is for the IV solar to be the hardest solar to make with LuV assembler, and HV with hardest EV, and so on. Is that unreasonable?

Whats the point? You can easily get multi amps of infinite IV from uranium alone.

**Comment by NeroOneTrueKing** [comment]
Currently:
```
Solar                     Tier    Requirements
1V, 8V                 --  MV     (general feasibility)
32V, 128V, 512V        --  EV     (IV circuits, sunnarium production)
2048V                  --  EV     (LuV circuits, EV circuit assembler, iridium processing, IV pumps for superconductors)
8192V                  --  IV     (ZPM circuits, IV circuit assembler)
```

0lafe proposes that solars should use one higher tier of circuits, thus needing one higher tier of circuit assembler:
```
Solar                     Tier    Requirements
1V, 8V                 --  MV     (general feasibility)
32V, 128V              --  EV     (IV circuits, HV circuit assembler, sunnarium production)
512V                   --  EV     (LuV circuits, EV circuit assembler)
2048V                  --  IV     (ZPM circuits, IV circuit assembler)
8192V                  --  LuV    (UV circuits, LuV circuit assembler, assembly line, fusion for europium for the UV circuits)
```

I dislike this idea because progressing to UV circuits needs both the assembly line and the fusion reactor, which limits pre-fusion solar power to EV.

What if, instead, the Sunnarium McGuffin each tier of solar needs must be assembled in a high tier assembling machine?
```
Solar                     Tier    Requirements
1V, 8V                 --  MV     (general feasibility)
32V, 128V              --  EV     (EV circuits, sunnarium production)
512V                   --  EV     (V circuits, EV assembler to make the "Sunnarium Alloy")
2048V                  --  IV     (LuV circuits, IV assembler to make the [newly added Sunnarium McGuffin to replace the 4x sunnarium plate])
8192V                  --  LuV    (ZPM circuits, LuV assembler to make the "Enriched Sunnarium Alloy", assembly line)
```

This way each tier of solar panel needs their tier+1, and there's a new solar available in between building the assembly line and building the fusion reactor.

**Comment by 0lafe** [comment]
>I dislike this idea because progressing to UV circuits needs both the assembly line and the fusion reactor, which limits pre-fusion solar power to EV.

So here's where I wanna throw in my solar experience. Limiting IV ones past fusion/assembly line, more shows the power of fusion than the weakness of solars that way. Locking IV's past fusion lets people make 4 EVs for 1A IV, or 16 EV or 1A LuV which is enough to do assembly line stuff and at least get an LuV naq gen or something.  Making 16 IVs was totally doable, so I would assume that making 16 EVs would also be very doable. 

> IV Solar    (ZPM circuits, LuV assembler to make the "Enriched Sunnarium Alloy", assembly line)

is the only gating difference between this and what I'm asking for, that the circuits I'm requesting cost europium which means fusion? If so, then I'm fully on board with another way of making it be put past the assembly line, and need something LuV'ey to make

**Comment by NeroOneTrueKing** [comment]
> is the only gating difference between this and what I'm asking for, that the circuits I'm requesting cost europium which means fusion?

Yup. It's a very long way between being able to make the pretty-decent EV solars and making your first fusion reactor.

**Comment by alphaNOVAircraft** [comment]
@noobyaran9 its 3 sunnarium plates plus 8 iridium reinforced plates.

**Comment by Prewf** [comment]
>This way each tier of solar panel needs their tier+1, and there's a new solar available in between building the assembly line and building the fusion reactor.

Really good suggestion, I like it. It keeps in line with the tech one has access to at those respective tiers, nothing too crazy, but not as easy as it is now (currently, IV solar doesn't even require an assembly line at all, even though you *should* have one already at this tier).

**Comment by Hakameda** [comment]
Maybe gated to something else that's in tier+1, instead of just circuits. Thing like LHE should always be better then passive power of the same tier or even +1

**Comment by Dream-Master** [comment]
Ok need to change quests as well.

**Comment by Logicus123** [comment]
Ok now the solars are exponentially expensiv; pretty hard to get them now

**Comment by Dream-Master** [comment]
orininal gt recipes using 4 solar pannels each tier so far i remember.

**Comment by Scottishprog** [comment]
I think that is an over-nerf... to the point solar cells won't be used at all. Having them as Tier +1 convenience generators would be OK. Thus, solars should never useful for advancing your max power production, but makes it easier to upgrade lesser things down the line. I don't think adding an exponential material cost is the way to go about this.

**Comment by Prewf** [comment]
im a little iffy on the doubling of solar panels every tier. i stand by the n+1 assembler and n+2 chip and the assembly line/fusion stuff; that stuff is basically making it conform to following a pattern

i.e. tech gate is good, but the material gate feels like maybe a bit too much

**Comment by noobyaran9** [comment]
If the nerf going to be doubling the number of solars required for the next tier, even though i hated it, I'd rather stick with @0lafe suggestion on the recipe.

>i.e. tech gate is good, but the material gate feels like maybe a bit too much

Basically this, tech gate is interesting if done right, but material gate is just plain boring. Excatly whats the end game and project ozone 2 kappa mode looks like.

**Comment by 0lafe** [comment]
It's not really a material gate until the super late tiers. But by then, needing like 512 LV circuits for LV solars isn't an issue, and it's half the circuits per tier. Barely any of the expensive circuits, and you should have insane autocrafting at that point. 


EV, the last solar before the quarry opens up, is just 8 LVs, and 16 Ulvs. Which seems reasonable for IV

**Comment by Prewf** [comment]
What if we didn't double the number of solars every tier? Keep in mind, Dream also doubled the number of circuits and superconductors at every tier of solar panel. Doubling the number of solar panels on top of that seems a bit heavy handed..

It's true what you said though: at that point you should have quarrying and AE2 set up. But also, you could honestly say that about any recipe at late tiers so I don't know how strong of an argument that is..

**Comment by sthegreat** [comment]
I agree that the exponential costs are unnecessary. We already have plenty of gating by the circuits, superconductors, alloys, etc. you need to progress through we dont need exponential material costs to balance things. We should balance by tech and complex setups because that is more interesting than just getting materials.

**Comment by 0lafe** [comment]
>so I don't know how strong of an argument that is..

Idea of solar is to be fully our of the tier of the solar, to be fully out of UV, being asked to craft like 256 LV solars, which is only 4 zpm, 16 Luv and 64IV, is more than reasonable. 

>e should balance by tech and complex setups because that is more interesting than just getting materials.

Big part of automating endgame is setting up stuff like parallel assembly lines, and a PA of EV+ machines for all singleblocks for crafting. Being asked to do what normally would be weeks of crafting, with a super lategame setup, is doable. 

>materials

Once you get the quarry, materials arent an issue, its all just processing times. Ore processing is so easy now because of gt++, hell, all lategame automation is to an extent. Need 1024 nanoprocessors? You'll always have the materials, and really should have the ability to make that amount fairly fast. My circuit assemblers were done very poorly, and it was still fast to make them.

**Comment by NeroOneTrueKing** [comment]
Note: Chromium Trioxide, which is needed for the plastic in the new higher-tier solars, appears to be unobtainable at the moment.

Not sure what else to contribute, other than to say that I hope to see other EV/IV methods of generating power to be added to the questbook and/or buffed. While this is a hardcore modpack, most nerfs (like this one) are hardest on new players trying to progress, and have the least impact on players already at endgame.

**Comment by Dream-Master** [comment]
@NeroOneTrueKing 
should be fixed in 2.0.4.7.
